templates/core/home.html

```diff
@@ -32,70 +32,41 @@ <h1 class="brand">{{ "cubes" | fas(fw=true) }} Docs.rs</h1>
-                    {%- if release.rustdoc_status -%}
-                        {%- set release_url = "/" ~ release.name ~ "/" ~ release.version ~ "/" ~ release.target_name ~ "/" -%}
-                    {%- else -%}
-                        {%- set release_url = "/crate/" ~ release.name ~ "/" ~ release.version -%}
-                    {%- endif -%}
+            {%- for release in recent_releases -%}
+                {%- if release.rustdoc_status -%}
+                    {%- set release_url = "/" ~ release.name ~ "/" ~ release.version ~ "/" ~ release.target_name ~ "/" -%}
+                {%- else -%}
+                    {%- set release_url = "/crate/" ~ release.name ~ "/" ~ release.version -%}
+                {%- endif -%}
 
-                    <li>
-                        <a href="{{ release_url | safe }}"
-                            class="release">
-                            <div class="pure-g">
-                                <div class="pure-u-1 pure-u-sm-6-24 pure-u-md-5-24 name">
-                                    {{ release.name }}-{{ release.version }}
-                                </div>
-                                <div class="pure-u-1 pure-u-sm-14-24 pure-u-md-16-24 description">
-                                    {{ release.description }}
-                                </div>
+                <a href="{{ release_url | safe }}"
+                    class="release">
+                    <div class="pure-g">
+                        <div class="pure-u-1 pure-u-sm-6-24 pure-u-md-5-24 name">
+                            {{ release.name }}-{{ release.version }}
+                        </div>
+                        <div class="pure-u-1 pure-u-sm-14-24 pure-u-md-16-24 description">
+                            {{ release.description }}
+                        </div>
 
-                                {%- if varsb.show_stars -%}
-                                    <div class="pure-u-1 pure-u-sm-4-24 pure-u-md-3-24 date">
-                                        {{ release.stars }}
-                                        {{ "star" | fas }}
-                                    </div>
-                                {%- else -%}
-                                    <div class="pure-u-1 pure-u-sm-4-24 pure-u-md-3-24 date"
-                                        title="{{ release.release_time | date(format='%FT%TZ') }}">
-                                        {{ release.release_time | timeformat(relative=true) }}
-                                    </div>
-                                {%- endif -%}
+                        {%- if varsb.show_stars -%}
+                            <div class="pure-u-1 pure-u-sm-4-24 pure-u-md-3-24 date">
+                                {{ release.stars }}
+                                {{ "star" | fas }}
                             </div>
-                        </a>
-                    </li>
-                {%- endfor -%}
-            </ul>
```

Comment:
Why did you remove these HTML tags?

Reply:
We cannot focus on a `<li>`, so I removed the `<ul>` and moved the `<a>` out.

Reply:
This seems like it might have the same accessibility problems as in https://github.com/rust-lang/rust/issues/87059 - is there another way you can do this?

Reply:
Hum... I don't know enough about accessibility to answer this. As for another way, I can focus on the `<a>` instead. So if you prefer for me to go this way, I'll update the PR to not change the DOM (or at least not change the current hierarchy).